.github/workflows/rill-cloud.yml

```diff
@@ -0,0 +1,52 @@
+# This workflow triggers deployment with Rill Cloud helm charts
+# Each merge to main branch is build with github sha tag and published to Rill Cloud.
+name: Deploy to Rill Cloud
+on:
+  push:
+    tags:
+      - "**"
+    branches:
+      - main
+      - cd
```

Comment:
Maybe just `main` branch (and tags are redundant, right?)

Reply:
we were planning to push main to `rill-deploy` workspace and tags to `margin-metering` or other co-dev workspaces.

Reply:
The co-dev workspaces should just use the normal releases/nightlies (it's basically just the local version in a pod). So shouldn't be mixed with the cloud deploys IMO.

The cloud deploys are multiple services and very differently designed/configured (like doing authentication in the application instead of in the ingress).